### DISCUSSION TITLE
"Sending each other SMS"

### DISCUSSION THREAD

**Discussion by hdsjulian** [egzumer/uv-k5-firmware-custom#474] [Ideas]
I want a cheap device that i can take to a festival (no internet) and also hand out to my friends and then send "hey i'm at location x" messages around. 
Is that something that would be implementable? If so: What kind of effort are we talking about?

**Comment by egzumer** [comment]
This firmware has messaging
https://github.com/joaquimorg/uv-k5-firmware-custom/tree/main/firmware

**Reply by hdsjulian** [reply]
amazing, thanks.
